cmd/argo/commands/lint.go

```diff
@@ -35,6 +35,9 @@ func NewLintCommand() *cobra.Command {
   cat manifests.yaml | argo lint --kinds=workflows,cronworkflows -`,
 		Run: func(cmd *cobra.Command, args []string) {
 			client.Offline = offline
+			if offline {
+				client.OfflineFiles = args
```

Comment:
I like the idea that all the files form a "linting context", do you even `OfflineFiles`?

Reply:
What do you mean?

Reply:
I think your contextual lint should be the default. No need to enable any flag. I think we can always do what line 39 does. This is always better and more correct.

Reply:
I removed the condition. It doesn't make sense indeed. The `OfflineFiles` are only used when the "offline client" is used